opensrp-giz-malawi/src/main/res/drawable/rounded_corner_edit_text_bg.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <solid android:color="#979797" />
```

Comment:
Move this reused color to the resources. The color is reused below in the same file.

Reply:
fixed